src/main/java/com/google/impactdashboard/servlets/GetProjectDataServlet.java

```diff
@@ -24,8 +24,10 @@
    */
   @Override
   public void init() {
+    try {
     projectInformationRetriever = ProjectInformationRetriever.create();
-  }
+    } catch (java.io.IOException e) {}
+   }
 
```

Comment:
We can just change this so that init throws the IOException. If we do that I think it will just pass the exception off to the error handler servlet.

Reply:
(we agreed offline that this would be handled in the next server PR)